### PR TITLE
Rename log in to sign in; Add reset password and register an account links

### PR DESCRIPTION
**Pull request by suricactus**
image

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-1802 QFieldSync add register button](https://app.clickup.com/t/2192114/QF-1802)

File: qfieldsync/ui/cloud_login_dialog.ui [hunk 1]
```diff
@@ -11,7 +11,7 @@
    </rect>
   </property>
   <property name="windowTitle">
-   <string>QFieldCloud Login Dialog</string>
+   <string>QFieldCloud Sign-in</string>
```
Comment (nirvn) [inline qfieldsync/ui/cloud_login_dialog.ui:14]: Sign-in or Sign in here?

**Comment by nirvn** [comment]
@suricactus , nice improvements (harmonization done and merged on the QField side of things too). 

I'm not a super fan of that divider line you added in the layout. The whole forgot?<br>------<br>register block is a bit dense. 

We could possibly hide the forgot password until someone fails to sign in once to make things look better. Thoughts?

File: qfieldsync/ui/cloud_login_dialog.ui (hunk 1 above)
Comment (suricactus) [reply qfieldsync/ui/cloud_login_dialog.ui:14]: Sign In

**Comment by suricactus** [comment]
Nice suggestion! 
image

**Comment by nirvn** [comment]
@suricactus , beautiful!

**Review by nirvn — APPROVED** [review]

**Pull request merged**